CODEOWNERS

```diff
@@ -23,7 +23,7 @@
 /components/CollectionCells/      @yarneo
 /components/CollectionLayoutAttributes/ @jverkoey
 /components/Collections/          @jverkoey
-/components/Dialogs/              @galiak11
+/components/Dialogs/              @galiak @iberezan
```

Comment:
This should be [@galiak11](https://github.com/galiak11).   @galiak is not on our team.